tests/test_multislice.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import raster_geometry
+from ioSPI import fourier, transfer
+from simSPI import (apply_complex_ctf_to_exit_wave, apply_dqe, apply_ntf,
+                    apply_poisson_shot_noise_sample, exit_wave_to_image)
+
+
+def test_exit_wave_to_image():
+    """High dose, no ctf/dqe/ntf."""
+    N = 64
+    sphere = raster_geometry.sphere([N, N, N], radius=N // 8, position=0.25).astype(
+        np.float32
+    )
+    ones = np.ones((N, N))
+    exit_wave = sphere.sum(-1)
+    exit_wave_f = fourier.do_fft(exit_wave, d=2)
+    high_dose = 1e9 * exit_wave.max()
+
+    i, shot_noise_sample, i0_dqe, i0 = exit_wave_to_image(
+        exit_wave_f=exit_wave_f,
+        complex_ctf=ones,
+        dose=high_dose,
+        noise_bg=0,
+        dqe=ones,
+        ntf=ones,
+    )
+
+    assert i.shape == [N, N]
+    assert shot_noise_sample.shape == [N, N]
+    assert i0_dqe.shape == [N, N]
+    assert i0.shape == [N, N]
+    assert np.allclose(i / high_dose, exit_wave, atol=1e-4)
+
+
+test_exit_wave_to_image()
```

Comment:
Is it necessary to execute the function explicitely?

Reply:
I'll remove that. I see you didn't do that in https://github.com/compSPI/ioSPI/blob/master/tests/test_iotools/test_atomic_models.py

I didn't make it a class...